day_6/exercises/person.rb

```diff
@@ -1,5 +1,31 @@
-# Create a person class with at least 2 attributes and 2 behaviors. 
+# Create a person class with at least 2 attributes and 2 behaviors.
 # Call all person methods below the class and print results
 # to the terminal that show the methods in action.
 
-# YOUR CODE HERE
+class Person
+  attr_reader :name, :age
+  def initialize (name, age)
+    @name = name
+    @age = age
+  end
+
+  def change_age(a)
+    age.replace(a)
+  end
+
+  def change_name(first)
+    name.replace(first)
+  end
+
```

Comment:
I don't see the `replace` method used all that much in these scenarios. I believe it works fine, but you can also just reassign your attributes.